packages/paste-core/components/form-pill-group/src/FormPillButton.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {pillStyles, hoverPillStyles} from './FormPill.styles';
+import type {PillVariant} from './types';
+
+interface FormPillStylesProps {
+  variant?: PillVariant;
+  selected?: boolean;
+  element?: string;
```

Comment:
Suggestion(non-blocking): for consistency, it might be better to use `element?: BoxElementProps['element']` here.